public/src/components/channelManagement/stickyTopBar/stickyTopBar.tsx

```diff
@@ -126,62 +142,72 @@ const StickyTopBar: React.FC<StickyTopBarProps> = ({
           </Button>
         </div>
       </div>
-      <div className={classes.lockContainer}>
-        {!userHasTestLocked && !lockStatus.locked && (
-          <>
-            <TestCopyButton
-              existingNames={existingNames}
-              existingNicknames={existingNicknames}
-              sourceName={name}
-              sourceNickname={nickname}
-              testNamePrefix={testNamePrefix}
-              onTestCopy={onTestCopy}
-              disabled={userHasTestListLocked}
-            />
-            <Button
-              variant="outlined"
-              size="medium"
-              startIcon={<EditIcon className={classes.icon} />}
-              onClick={() => onTestLock(name, false)}
-            >
-              <Typography className={classes.buttonText}>Edit test</Typography>
-            </Button>
-          </>
-        )}
-        {!userHasTestLocked && lockStatus.locked && (
-          <>
-            <TestLockDetails email={lockStatus.email} timestamp={lockStatus.timestamp} />
-            <Button
-              variant="outlined"
-              size="medium"
-              startIcon={<LockIcon className={classes.icon} />}
-              onClick={() => onTestLock(name, true)}
-            >
-              <Typography className={classes.buttonText}>Take control</Typography>
-            </Button>
-          </>
-        )}
-        {userHasTestLocked && (
-          <>
-            {!isNew && <TestArchiveButton onTestArchive={onTestArchive} />}
-            <Button
-              variant="outlined"
-              size="medium"
-              startIcon={<CloseIcon className={classes.icon} />}
-              onClick={() => onTestUnlock(name)}
-            >
-              <Typography className={classes.buttonText}>Discard</Typography>
-            </Button>
-            <Button
-              variant="outlined"
-              size="medium"
-              startIcon={<SaveIcon className={classes.icon} />}
-              onClick={() => onTestSave(name)}
-            >
-              <Typography className={classes.buttonText}>Save test</Typography>
-            </Button>
-          </>
-        )}
+
+      <div className={classes.buttonsContainer}>
```

Comment:
this change moves all the ctas into the `buttonsContainer` div, and adds the `TestLiveSwitch`